.github/workflows/tests.yml

```diff
@@ -40,11 +40,6 @@ jobs:
           # PHP 7.1 development web server segfaults if timezone not set.
           ini-values: date.timezone=Europe/Paris, error_reporting=-1, display_errors=On
 
-      - name: Configure for PHP >= 7.1
-        if: "${{ matrix.php >= '7.1' && matrix.symfony_version == '' }}"
-        run: |
-          composer require --no-update --dev symfony/error-handler "^4.4 || ^5.0"
-
```

Comment:
@stof , isn't this needed anymore? The condition is exactly for the PHP versions in use.

Reply:
As we don't need to support PHP <7.1 anymore, this dev requirement is now defined in the composer.json file directly instead of being added dynamically by the CI

Reply:
I see. Thank you.